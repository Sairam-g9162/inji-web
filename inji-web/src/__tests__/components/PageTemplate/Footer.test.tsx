import React from 'react';
import {render, screen} from '@testing-library/react';
import {Footer} from "../../../components/PageTemplate/Footer";
describe("Footer Container",() => {
    test('check the presence of the container', () => {
        const {asFragment} = render(<Footer />);
        expect(asFragment()).toMatchSnapshot();
    });
    
})
describe("Test the functionality of the Footer Container",()=>{
    test('check the presence of the container', () => {
        render(<Footer />);
        const footerElementText = screen.getByTestId("Footer-Text");
        expect(footerElementText).toHaveTextContent("Footer.copyRight");
    });
})

