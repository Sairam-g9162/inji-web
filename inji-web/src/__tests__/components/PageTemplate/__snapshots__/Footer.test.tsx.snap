// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer Container check the presence of the container 1`] = `
<DocumentFragment>
  <footer
    class="fixed bottom-0 left-0 right-0 py-4 text-center transform rotate-180 shadow-sm shadow-iw-shadow bg-iw-footer"
    data-testid="Footer-Container"
  >
    <div
      class="container mx-auto"
    >
      <p
        class="text-black transform rotate-180"
        data-testid="Footer-Text"
      >
        Footer.copyRight
      </p>
    </div>
  </footer>
</DocumentFragment>
`;
